tests/v2/test_init.py

```diff
@@ -0,0 +1,30 @@
+"""Test v2 bridge."""
+from pathlib import Path
+from unittest.mock import patch
+
+from aiohue import HueBridgeV2
+
+from tests.common import load_fixture
+
+
+async def test_bridge_init():
+    """Test v2 bridge."""
+    bridge = HueBridgeV2("192.168.1.123", "mock-key")
+    assert bridge.host == "192.168.1.123"
+
+    with patch.object(
+        bridge, "request", return_value=load_fixture("v2_resources.json")
```

Comment:
Consider moving the loading of the data from disk to a fixture that is only run once per session. A side effect benefit of that is also that we don't block the the event loop in the middle of this test.